### PR TITLE
[chore] make update-otel

### PR DESCRIPTION
**Pull request by codeboten**
Signed-off-by: Alex Boten <aboten@lightstep.com>

### CODE REVIEW

File: cmd/configschema/go.mod [hunk 1]
```diff
@@ -5,12 +5,12 @@ go 1.18
 require (
 	github.com/fatih/structtag v1.2.0
 	github.com/google/uuid v1.3.0
-	github.com/open-telemetry/opentelemetry-collector-contrib v0.67.0
+	github.com/open-telemetry/opentelemetry-collector-contrib v0.0.0-00010101000000-000000000000
```
Comment (MovieStoreGuy) [inline cmd/configschema/go.mod:8]: This should be using the version right? not the version alias used by `replace local` ?

File: internal/otelcontribcore/main.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ import (
 	"log"
 
 	"go.opentelemetry.io/collector/component"
-	"go.opentelemetry.io/collector/service"
+	"go.opentelemetry.io/collector/otelcol"
```
Comment (MovieStoreGuy) [inline internal/otelcontribcore/main.go:25]: Was this a deprecation change?

File: receiver/receivercreator/runner.go [hunk 3]
```diff
@@ -64,7 +64,6 @@ func (run *receiverRunner) start(
 
 	// Sets dynamically created receiver to something like receiver_creator/1/redis{endpoint="localhost:6380"}/<EndpointID>.
 	id := component.NewIDWithName(factory.Type(), fmt.Sprintf("%s/%s{endpoint=%q}/%s", receiver.id.Name(), run.idNamespace, endpoint, receiver.endpointID))
-	cfg.SetIDName(id.Name()) //nolint:staticcheck
```
Comment (MovieStoreGuy) [inline receiver/receivercreator/runner.go:67]: Another deprecation?

File: cmd/configschema/go.mod (hunk 1 above)
Comment (MovieStoreGuy) [inline cmd/configschema/go.mod:8]:
```suggestion
	github.com/open-telemetry/opentelemetry-collector-contrib v0.67.0
```
I believe?

File: internal/otelcontribcore/main.go (hunk 2 above)
Comment (codeboten) [reply internal/otelcontribcore/main.go:25]: Yes, still working through more of the changes

**Comment by codeboten** [comment]
Closing in favour of smaller PRs starting with https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/17032

**Pull request closed without merging**